### PR TITLE
Removed "browser" references from "it" blocks

### LINKED ISSUE #5677: Remove references to browser from 'it' blocks
All browser refs should be in helpers/pageobjects.

### PR DESCRIPTION
**Pull request by MartinSchoeler**
And remove some unnecessary env vars handling
closes #5677

### CODE REVIEW

File: tests/end-to-end/ui/06-messaging.js [hunk 1]
```diff
@@ -199,7 +199,7 @@ function messageActionsTest() {
 				it('delete the message', () => {
 					mainContent.selectAction('delete');
 					mainContent.popupFileConfirmBtn.click();
-					browser.waitForVisible('.sweet-overlay', 3000, true);
+					flexTab.sweetAlertOverlay.waitForVisible(3000, true);
```
Comment (rodrigok) [inline tests/end-to-end/ui/06-messaging.js]: `sweetAlertOverlay` could be part of `page` object don't you think? Cuz it's more global then related with flexTab component.

File: tests/end-to-end/ui/13-permissions.js [hunk 2]
```diff
@@ -98,9 +98,7 @@ describe('Admin settings', () => {
 		});
 
 		it('should show the user in the list', () => {
-			browser.pause(200);
-			var element = browser.element('td=adminCreated'+username);
-			element.isVisible().should.be.visible;
+			admin.checkUserList(username).should.contain(true);
```
Comment (rodrigok) [inline tests/end-to-end/ui/13-permissions.js]: Why `contain(true)` rather than `should.be.true`?

File: tests/end-to-end/ui/06-messaging.js (hunk 1 above)
Comment (MartinSchoeler) [inline tests/end-to-end/ui/06-messaging.js]: Yes that's a good idea, do you think i should change some more global elements, like toast, to page?

File: tests/end-to-end/ui/13-permissions.js (hunk 2 above)
Comment (MartinSchoeler) [inline tests/end-to-end/ui/13-permissions.js]: I was checking something, also i forgot to add the in progress label in this pr

File: tests/end-to-end/ui/06-messaging.js (hunk 1 above)
Comment (rodrigok) [inline tests/end-to-end/ui/06-messaging.js]: Yeap, that sounds good

**Pull request merged**